### ISSUE TITLE
Wan 2.2 training on high AND low noise generated only one lora

### ISSUE THREAD

**Issue by BenDes21** [kohya-ss/musubi-tuner#569]
Hi there,

Im trying to train Wan 2.2 i2v on both low and high model, here my config  :   

accelerate launch --num_cpu_threads_per_process 1 --mixed_precision fp16 src/musubi_tuner/wan_train_network.py \
    --max_train_epochs 30 --save_every_n_epochs 1 --seed 42 \
    --output_dir /workspace/output --output_name motionv1 \
    --network_module networks.lora_wan \
    --task i2v-A14B \
    --dataset_config /workspace/dataset.toml \
    --dit /workspace/models/wan2.2_i2v_low_noise_14B_fp16.safetensors \
    --dit_high_noise /workspace/models/wan2.2_i2v_high_noise_14B_fp16.safetensors \
    --t5 /workspace/models/umt5-xxl-enc-bf16.safetensors \
    --vae /workspace/models/wan_2.1_vae.safetensors \
    --discrete_flow_shift 5.0 --timestep_sampling shift --timestep_boundary 900 \
    --gradient_checkpointing --rope_func comfy --sdpa --preserve_distribution_shape  \
    --fp8_base --fp8_scaled --mixed_precision fp16 \
    --max_data_loader_n_workers 16 --persistent_data_loader_workers \
    --network_module networks.lora_wan --network_dim 16 --network_alpha 16 --network_args loraplus_lr_ratio=4 \
    --optimizer_type schedulefree.RAdamScheduleFree --learning_rate 0.0002

Im not sure why it's generated only 1 lora instead of 2 ( low and high noise ) and also not sure why Im getting a progress on /300 instead of 30 ( 30 epochs / 1 saved per epoch ) 

<img width="458" height="48" alt="Image" src="https://github.com/user-attachments/assets/3706c6cb-908b-4c17-b41e-2579de86f554" />

Probably a config mistake

**Comment by kohya-ss** [comment]
If you specify both `--dit` and `--dit_high_noiseP, one LoRA that can be applied to both high and low models will be trained. It is not possible to train two LoRAs at the same time, so train one LoRA for high and one for low separately. See the documentation for details: https://github.com/kohya-ss/musubi-tuner/blob/main/docs/wan.md#training

The progress bar shows the number of training steps. This is because your dataset requires 10 steps of training per epoch.

**Comment by BenDes21** [comment]
Hi there ! 

Thanks for your answer, there is no difference of result when :  
 
- The lora trained with both model is applied on both low and high noise ( comfyUI workflow )
- 2 lora's trained separately ( high then low let's said ) are applied on 2 different samplers  

right ?

**Comment by kohya-ss** [comment]
I'm still not sure, but there are probably the following differences between the two: A single LoRA applied to both high and low models is trained on all timesteps, so it will learn motion, details, etc. On the other hand, the accuracy specialized in one area may be inferior to two LoRAs.

If separate LoRAs are trained for high and low, the timesteps of each LoRA will be different, so the LoRA for high will learn composition and motion, and the LoRA for low will learn details.

**Comment by BenDes21** [comment]
just getting terrible results with this config, Im using schedulefree.RAdamScheduleFree which supposed to help but I dont know its give very very bad render

accelerate launch --num_cpu_threads_per_process 1 --mixed_precision fp16 src/musubi_tuner/wan_train_network.py \
    --max_train_epochs 50 --save_every_n_epochs 1 --seed 42 \
    --output_dir /workspace/output --output_name motionv1 \
    --network_module networks.lora_wan \
    --task i2v-A14B \
    --dataset_config /workspace/dataset.toml \
    --dit /workspace/models/wan2.2_i2v_low_noise_14B_fp16.safetensors \
    --dit_high_noise /workspace/models/wan2.2_i2v_high_noise_14B_fp16.safetensors \
    --t5 /workspace/models/umt5-xxl-enc-bf16.safetensors \
    --vae /workspace/models/wan_2.1_vae.safetensors \
    --discrete_flow_shift 5.0 --timestep_sampling shift --timestep_boundary 900 \
    --gradient_checkpointing --rope_func comfy --sdpa --preserve_distribution_shape  \
    --fp8_base --fp8_scaled --mixed_precision fp16 \
    --max_data_loader_n_workers 16 --persistent_data_loader_workers \
    --network_module networks.lora_wan --network_dim 16 --network_alpha 16 --network_args loraplus_lr_ratio=4 \
    --optimizer_type schedulefree.RAdamScheduleFree --learning_rate 0.002

**Comment by Sarania** [comment]
I don't think you need `--preserve_distribution_shape` when training them together, only when training them one at a time though I don't /think/ it will hurt anything. Your LR is also way too high for doing dim=alpha especially with LoraPlus so set your LR to something like 0.00002 (2e-5) if you wanna do dim=alpha, or set alpha=1 and try an LR like 0.0002 (2e-4). And lastly while I don't think it would hurt anything, `--rope_func comfy` won't really benefit you when you're not using compile.
